spec/support/stubbed_vacuum.rb

```diff
@@ -0,0 +1,68 @@
+class StubbedVacuum
+  def initialize(details)
+    @details = details
+  end
+
+  def associate_tag=(something)
+    #noop
+  end
+
+  def item_lookup(query)
+    VacuumResponse.new(@details)
+  end
+end
+
+class VacuumResponse
+  def initialize(details)
+    @details = details
+  end
+
+  def to_h
+    {"ItemLookupResponse"=>
+     {"OperationRequest"=>
+      {"HTTPHeaders"=>{"Header"=>{"Name"=>"UserAgent", "Value"=>"Jeff/1.5.1 (Language=Ruby; ares.local)"}},
+       "RequestId"=>"a8279e28-02d2-4703-b4bc-669ab41f4d02",
+       "Arguments"=>
+      {"Argument"=>
+       [
+         {"Name"=>"AWSAccessKeyId", "Value"=>"AKIAINQWKZ76LYIIEWVQ"},
+         {"Name"=>"Signature", "Value"=>"/MUlX8F8H8bsHQJmvzsy3XPiVEUwjmEyPV3W0tScT3Q="}]},
+      "RequestProcessingTime"=>"0.0088050000000000"},
+      "Items"=>
+      {"Request"=>{"IsValid"=>"True", "ItemLookupRequest"=>{"IdType"=>"ASIN", "ItemId"=>"B004UJ0U0C", "ResponseGroup"=>"ItemAttributes", "VariationPage"=>"All"}},
+       "Item"=>
+      {"ASIN"=>"B004UJ0U0C",
+       "ParentASIN"=>"B004UJ0TDU",
+       "DetailPageURL"=> @details[:url],
+       "ItemLinks"=>
+      {"ItemLink"=>
+       [{"Description"=>"All Offers", "URL"=>"http://www.amazon.com/gp/offer-listing/B004UJ0U0C%3FSubscriptionId%3DAKIAINQWKZ76LYIIEWVQ%26tag%3Dbestow-20%26linkCode%3Dxm2%26camp%3D2025%26creative%3D386001%26creativeASIN%3DB004UJ0U0C"}]},
+      "ItemAttributes"=>
+       {"Binding"=>"Kitchen",
+        "Brand"=>"Simple Ecology",
+        "Color"=>"Natural Cotton",
+        "EAN"=>"0753182636542",
+        "EANList"=>{"EANListElement"=>"0753182636542"},
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.